src/index.ts

```diff
@@ -0,0 +1,34 @@
+import { IOBuffer } from 'iobuffer';
+
+import { readDataBlock, Spectrum } from './dataBlock';
+import { LogBlock, readLogBlock } from './logBlock';
+import { Header, mainHeader } from './mainHeader';
+
+declare type InputData =
+  | number
```

Comment:
Number should not be allowed. It allows to create a IOBuffer from scratch but here we must have data.

Reply:
I edited the type in the last commit